test/distributed/test_functional_api.py

```diff
@@ -781,6 +781,44 @@ def test_dead_wrapper_plus_view(self):
         res = None
         self.assertEqual(0, ft_c_impl._outstanding_wait_count())
 
+@instantiate_parametrized_tests
+class TestFunctionalAutograd(MultiThreadedTestCase):
+    def setUp(self):
+        super().setUp()
+        self._spawn_threads()
+
+    @property
+    def world_size(self):
+        return 2
+
+    @parametrize("compile", [True, False])
+    def test_all_to_all_single(self, compile: bool = True) -> None:
+        group = "0"
```

Comment:
nit: group names are subject to change. It's better to get it from `dist.group.WORLD` or just pass `dist.group.WORLD` to the funcol API.